stdlib/public/core/Integers.swift.gyb

```diff
@@ -424,46 +1729,202 @@ public func > <T : BinaryInteger>(lhs: T, rhs: T) -> Bool {
 //===--- FixedWidthInteger ------------------------------------------------===//
 //===----------------------------------------------------------------------===//
 
+/// An indicator of whether an arithmetic operation overflowed.
+///
+/// Some arithmetic operations on fixed-width integers return an
+/// `ArithmeticOverflow` instance to indicate whether an overflow has
+/// occurred. For example, adding `UInt8.max` to itself results in a value that
+/// can't be represented by an `UInt8` instance without overflowing.
+///
+///     let x = UInt8.max
+///     // x == 255
+///     let (y, overflow) = x.addingWithOverflow(x)
+///     // y == 254
+///     // overflow == ArithmeticOverflow.overflow
 public enum ArithmeticOverflow {
   @_transparent
   public init(_ overflow: Bool) { self = overflow ? .overflow : .none }
-  case none, overflow
+  /// An indication that no overflow occured in the operation.
+  case none
+  /// An indication that an overflow did occur in the operation.
+  case overflow
 }
 
+/// An integer type that uses a fixed size for every instance.
+///
+/// The `FixedWidthInteger` protocol adds binary bitwise operations, bit
+/// shifts, and overflow handling to the operations supported by the
+/// `BinaryInteger` protocol.
+///
+/// Use the `FixedWidthInteger` protocol as a constraint or extension point
+/// when writing operations that depend on bit shifting, performing bitwise
+/// operations, catching overflows, or having access to the maximum or minimum
+/// representable value of a type. For example, the following code provides a
+/// `binaryString` property on every fixed-width integer that represents the
+/// number's binary representation, split into 8-bit chunks.
+///
+///     extension FixedWidthInteger {
+///         var binaryString: String {
+///             var result: [String] = []
+///             for i in Swift.stride(from: 0, to: Self.bitWidth, by: 8) {
```

Comment:
It does—I think it's running into the deprecation warnings here: https://github.com/apple/swift/blob/master/stdlib/public/core/Stride.swift.gyb#L305-L318